src/pages/staking/StakeForm.vue

```diff
@@ -271,8 +271,19 @@ export default {
                         this.bottomInputAmount = '';
                         console.error(`Unable to convert TLOS to STLOS: ${err}`);
                     })
-                    .finally(() => {
-                        return this.bottomInputIsLoading = false;
+                    .finally(async () => {
+                        this.bottomInputIsLoading = false;
+
+                        if (getClientIsApple()) {
+                            // workaround; Apple devices will focus the last input which had its value
+                            // programmatically changed. Focus should not change. Downside is that cursor position
+                            // is lost, but generally cursor should be at the end anyway
+                            const old = this.topInputAmount;
+                            this.topInputAmount = '0';
+                            await this.$nextTick();
+
+                            this.topInputAmount = old;
+                        }
```

Comment:
Could we turn `this.topInputAmount` and `this.bottomInputAmount` into parameters and reuse the function? I think there are 4 places we could use such a function

Reply:
these functions rely on the `this` context and also differ slightly each time (depending on whether it needs top or bottom input), i dont think it'll be worth refactoring because of that complexity